backend/src/main/java/ca/gov/dtsstn/cdcp/api/service/UserService.java

```diff
@@ -197,6 +205,19 @@ public boolean verifyEmail(String userId, String code){
 		return true;
 	}
 
+	public User createUser(String userEmail, List<UserAttributeModel> userAttributesModel){
+		//Verify Email
+		Assert.hasText(userEmail, "userEmail is required; it must not be null or blank");
+
+		Collection<UserAttributeEntity> userAttributesCollection = new ArrayList<UserAttributeEntity>();
+		
+		for(UserAttributeModel e : userAttributesModel){
+			userAttributesCollection.add(new UserAttributeEntityBuilder().name(e.getName()).value(e.getValue()).build());
+		}
```

Comment:
This code should be removed once the method is updated to accept a `User` instance.